groups/infrastructure/locals.tf

```diff
@@ -0,0 +1,38 @@
+locals {
+  stack_name     = "file-transfer"
+  stack_fullname = "${local.stack_name}-stack"
+  name_prefix    = "${local.stack_name}-${var.environment}"
+
+  stack_secrets = jsondecode(data.vault_generic_secret.secrets.data_json)
+
+  application_subnet_pattern  = local.stack_secrets["application_subnet_pattern"]
+  public_subnet_pattern       = local.stack_secrets["public_subnet_pattern"]
+  application_subnet_ids      = join(",", data.aws_subnets.application.ids)
+  kms_key_alias               = local.stack_secrets["kms_key_alias"]
+  vpc_name                    = local.stack_secrets["vpc_name"]
+  notify_topic_slack_endpoint = local.stack_secrets["notify_topic_slack_endpoint"]
+
+  parameter_store_secrets = {
+    "vpc-name"                 = local.stack_secrets["vpc_name"],
+    "web-oauth2-client-id"     = local.stack_secrets["web-oauth2-client-id"],
+    "web-oauth2-client-secret" = local.stack_secrets["web-oauth2-client-secret"],
+    "web-oauth2-cookie-secret" = local.stack_secrets["web-oauth2-cookie-secret"],
+    "web-oauth2-request-key"   = local.stack_secrets["web-oauth2-request-key"]
+  }
+
+  application_ids   = data.aws_subnets.application.ids
+  application_cidrs = [for s in data.aws_subnet.application : s.cidr_block]
+  public_ids        = data.aws_subnets.public.ids
+  public_cidrs      = [for s in data.aws_subnet.public : s.cidr_block]
+
+  public_lb_cidrs    = ["0.0.0.0/0"]
```

Comment:
This needs examining, even with an internal lb I don't think we want to expand to public cidrs